.github/workflows/smoke-test-package.yaml

```diff
@@ -0,0 +1,236 @@
+name: Smoke test package rpm/deb
+run-name: Smoke test package rpm/deb
+
+on:
+  workflow_dispatch:
+    inputs:
+      release-version:
+        description: |
+          Override the build version from what is in the Makefile.
+          Use this override to specify a tag for build debugging purposes.
+        type: string
+
+  # TODO(vapopov): delete before merge.
+  push:
+    branches:
+      - vapopov/smoke-test-package
+
+  merge_group:
+    paths:
+      - 'lib/autoupdate/**.go'
+      - 'tool/teleport-update/**.go'
+      - 'go.mod'
+      - 'go.sum'
+      - 'build.assets/Makefile'
+      - 'build.assets/Dockerfile*'
+      - 'Makefile'
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.set-version.outputs.version }}
+    env:
+      # GOMODCACHE and GOCACHE can be removed when we commit to the self-hosted
+      # ARC runners, as these runners set up these vars for us. We still need
+      # these though for the GitHub runners (ubuntu).
+      GOMODCACHE: /tmp/gomodcache
+      GOCACHE: /tmp/gocache
+
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2 # v3.10.0
+        with:
+          driver: docker
+
+      - name: Checkout Teleport
+        uses: actions/checkout@v4
+
+#      # Log into Docker Hub to increase rate limits
+#      - name: Login to Docker Hub
+#        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
+#        with:
+#          username: ${{ secrets.DOCKERHUB_USERNAME }}
+#          password: ${{ secrets.DOCKERHUB_TOKEN }}
+#
+#      # Log into ECR to raise rate limits, needed for the public.ecr.aws/gravitational/fpm
+#      # image used for building .deb and .rpm files.
+#      - name: Log into ECR
+#        uses: aws-actions/amazon-ecr-login@062b18b96a7aff071d4dc91bc00c4c1a7945b076 # v2.0.1
+#        with:
+#          registry-type: public
+#        env:
+#          # GetAuthorizationToken command is only supported in us-east-1.
+#          AWS_REGION: us-east-1
```

Comment:
```suggestion
```

Reply:
It's commented out because we don't have those secrets in the teleport repo (or there were incorrect request). Should we include this login? There was one failure due to rate limiting during my testing

Reply:
Ah I see. Yes, we will need to do something here to handle rate limits, especially with dockerhub.